crates/sparrow-session/src/session.rs

```diff
@@ -43,6 +43,13 @@ pub enum Results {
     Snapshot,
 }
 
+#[derive(Debug, PartialEq, Eq)]
+pub enum QueryPlanKind {
```

Comment:
Maybe `QueryRepresentationKind`? Is a `dfg` a type of `QueryPlan`? 

I ask because the `FInalPlan` would correspond to the `ComputePlan` so just want to clarify naming here.

Reply:
I'm considering for now the `dfg` to be approximately a logical plan. Once the DFG goes away, then the equivalent would be choosing between rendering the logical plan and the physical plan.

Reply:
Went with `ExplanationKind`